src/server/game/Entities/Pet/Pet.cpp

```diff
@@ -948,6 +949,22 @@ bool Guardian::InitStatsForLevel(uint8 petlevel)
             //damage range is then petlevel / 2
             SetBaseWeaponDamage(BASE_ATTACK, MAXDAMAGE, float(petlevel + (petlevel / 4)));
             //damage is increased afterwards as strength and pet scaling modify attack power
+            if (Player* player = ToPlayer())
+            {
+                if (Pet* pet = player->GetPet())
+                {
+                    if (pet->GetAura(SPELL_WILD_HUNT_R2))
+                    {
+                        SetModifierValue(UNIT_MOD_STAT_STAMINA, BASE_VALUE, float(m_owner->GetStat(STAT_STAMINA)) * 0.4f);
+                        SetBonusDamage(int32(GetOwner()->GetTotalAttackPowerValue(BASE_ATTACK) * 0.3f));
```

Comment:
Yes, line 959 and 964 needs the same info source as StatSystem.cpp line 1310 (R.A.P.).